custom/icds_reports/reports/new_born_with_low_weight.py

```diff
@@ -228,6 +233,9 @@ def get_newborn_with_low_birth_weight_data(domain, config, loc_level, show_test=
 
         value = (low_birth or 0) * 100 / float(in_month or 1)
 
+        tooltips_data[name]['low_birth'] += low_birth
```

Comment:
in line 234 there is a line (low_birth or 0) does this need or 0 as well?

Reply:
Yes this is needed, I changed this in line 232